ansible/fgdc2iso.yml

```diff
@@ -43,3 +43,53 @@
   hosts: catalog-fgdc2iso
   roles:
     - role: software/datagov-deploy-fgdc2iso
+
+  # Host-level smoke tests
+  tasks:
+    - name: flush handlers
+      meta: flush_handlers
+
+    - name: assert app is up
+      uri:
+        url: http://localhost/fgdc2iso/
+        follow_redirects: all
+        method: POST
+        body: |
+          <metadata>
+            <idinfo>
+              <citation>
+                <citeinfo>
+                  <title>ABC 123</title>
+                </citeinfo>
+              </citation>
+            </idinfo>
+          </metadata>
+        status_code: 200
+        return_content: yes
+        # TODO enable cert validation. Staging and production hosts have GSA
+        # signed certs which should be valid.
+        # https://github.com/GSA/datagov-deploy/issues/900
+        validate_certs: false
+      retries: 3
+      delay: 10
+      register: result
+      until: not result.failed
+
+    - name: assert valid iso returned
+      assert:
+        that: "'<gco:CharacterString>ABC 123' in result.content"
```

Comment:
[According to the docs](https://docs.ansible.com/ansible/latest/modules/assert_module.html), it looks like `that` should be a list. Since this is a string, it looks like you're iterating a string, which is always truthy, thus a false positive test assertion. Maybe instead:

```yaml
- name: assert valid iso returned
  assert: '<gco:CharacterString>ABC 123' in result.content

Reply:
@adborden Nope, without `that` it is invalid syntax. When only one value, it is ok to use string as `that`'s value, same as the first example given in the [doc](https://docs.ansible.com/ansible/latest/modules/assert_module.html). 
`- assert: { that: "ansible_os_family != 'RedHat'" }`